pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -68,6 +68,9 @@ def configure_ddp(self, model, device_ids):
             device_ids=device_ids,
             **self._ddp_kwargs,
         )
+        if model.module.decision_on_invalid_result == DecisionOnInvalidResult.NEVER_SKIP:
```

Comment:
why do we need to set these here if it is already done in `_backward_with_possible_nan_loss`?

Reply:
If I don't, it hangs in DDP.

Reply:
Definitely need more investigation 👍

Reply:
Seemed to work when removing then.